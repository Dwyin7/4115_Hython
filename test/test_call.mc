def int test_call_1 () {
    int i = 1;
}

def int test_call_2 () {
    int call1 = test_call_1();
    int res = call1 + 2;
}

def float test_call_3 () {
    int call2 = test_call_2();
    int res = call2 + 3;
}

def void test_call_4() {
    int i;
}

test_call_1();
test_call_2();
test_call_3();
test_call_4();


