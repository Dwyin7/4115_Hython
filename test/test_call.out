import 1 as 2
func int test_call_1() {
int i = (int) 1
}
func int test_call_2() {
int call1 = (int) test_call_1()
int res = (int) (int) call1 + (int) 2
}
func float test_call_3() {
int call2 = (int) test_call_2()
int res = (int) (int) call2 + (int) 3
}
func void test_call_4() {
int i
}
(int) test_call_1()
(int) test_call_2()
(float) test_call_3()
(void) test_call_4()
